CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+# v68.0.0 (_2020-12-08_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v67.1.0...v68.0.0)
+
+## General
+
+### ⚠️ Breaking changes ⚠️
+
+- The bundled version of Nimbus SDK has been updated to v0.6.1, which includes
+  the following breaking changes:
+  - Removed `NimbusClient.resetEnrollment`.
+  - `NimbusClient.{updateExperiments, optInWithBranch, optOut, setGlobalUserParticipation}` now return a list of telemetry events.
+    Consumers should forward these events to their telemetry system (e.g. via Glean).
+  - Removed implicit fetch of experiments on first use of the database. Consumers now must
+    call update_experiments explicitly in order to fetch experiments from the Remote Settings
+    server.
+
+
+### What's Changed
+
+- The bundled version of Glean has been updated to v33.5.0.
```

Comment:
The latest glean is `v33.7.0`, but trying to build that errors out with warnings about an unknown `#![deny(broken_intra_doc_links)]` lint, and I've collected enough yak hair from this release already.